### ISSUE TITLE
Carousel view

### ISSUE THREAD

**Issue by aloscha** [StarcounterPrefabs/Images#55]
After we start support carousel view (it means we could see multiple images) we should have possibility to add mutliple images from basic app, for now it's unavailable - I could drop just one image.

**Comment by warpech** [comment]
@diliger, can this issue be closed now?

**Comment by aloscha** [comment]
This working only if someone used partial. I think `Carousel view` should be possible get from standalone app.

**Comment by diliger** [comment]
@warpech, should I improve the `juicy-filedrop` element to accept multiple file drop?

**Comment by aloscha** [comment]
I think it will be cool. If you add mutliple files than you have carousel view, if just one you'll have just one image/media.

IMHO:
1. Improve juicy-filedrop to handle multiple files (what if one of them will be image and one will be video? Does we support that situation? @warpech )
2. After added couple images we should have carousel view

**Comment by warpech** [comment]
> If you add mutliple files than you have carousel view, if just one you'll have just one image/media.

Then how do you add the second image?

By dragging a new image on top of the existing image? I think that should replace the existing image, not add a new one.

---

I think that thinking of `/images/partials/somethings/{Something ObjectID}` as a **carousel view** is wrong.

We should think of it a collection of illustrations of something.

Carousel is just a single view, but we can add more views in future:

- Masonry
![image](https://cloud.githubusercontent.com/assets/566463/23798498/3a4fbad2-05a5-11e7-85d3-5a726f712f1c.png)
- Swipe left & right
![image](https://cloud.githubusercontent.com/assets/566463/23798582/99367964-05a5-11e7-89be-abd8efc58050.png)
- Explorer
![image](https://cloud.githubusercontent.com/assets/566463/23798604/b54c77b6-05a5-11e7-9f67-79da3ff3764b.png)

**Comment by aloscha** [comment]
>>If you add mutliple files than you have carousel view, if just one you'll have just one image/media.

>Then how do you add the second image?

If you add from disk couple images at the same time. (I mean selected few images in dialog)

**Comment by warpech** [comment]
But let's say you have added an image yesterday and now you want to add a second one. Where is the "Add new image" button then, if there is no carousel?

**Comment by aloscha** [comment]
Agreed. So maybe let's add button - `Add another button` (and multi adding should be available from settings) WDYT?

**Comment by warpech** [comment]
I don't think it is a matter of global settings. This a matter of uri mapping. 

- If you want one image, map the partial in your app to `/images/partials/somethings-single-edit/{Something ObjectID}` (**which does not exist now**)
- If you want one or more images, map the partial in your app to `/images/partials/somethings-edit/{Something ObjectID}`
